File: EmprestimoDVD-API/EmprestimoDVD.Domain/Entidades/DVD.cs
﻿namespace EmprestimoDVD.Domain.Entidades
{
    public class DVD : BaseEntity
    {
        public string Titulo { get; set; }
        public string Sinopse { get; set; }
        public FaixaEtaria Classificacao { get; set; }
        public Pessoa ArtistaPrincipal { get; set; }
        public Pessoa Diretor { get; set; }
        public Genero Genero { get; set; }
    }
}

File: EmprestimoDVD-Front/src/app/demo/api/dvd.ts
export interface dvd {
    id?: number;
    titulo?: string;
    sinopse?: string;
    idadeMinima?: number;
    idArtistaPrincipal?: number;
    idDiretor?: number;
    idGenero?: number;
}

export interface dvdGet {
    id?: number;
    titulo?: string;
    sinopse?: string;
    idadeMinima?: number;
    artistaPrincipal?: {
        nome: string;
        id: number;
    };
    diretor?: {
        nome: string;
        id: number;
    };
    genero?: {
        descricao: string;
        id: number;
    };
}
